.github/workflows/install_ci.yml

```diff
@@ -12,33 +12,25 @@ jobs:
   test:
     strategy:
       matrix:
-        include:
-          - os: ubuntu-20.04
-          - os: macos-latest
+        os: [ubuntu-20.04, macos-latest, windows-latest]
+      fail-fast: false
 
     name: Install v-analyzer on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    continue-on-error: true
 
     steps:
-      - uses: msys2/setup-msys2@v2
-        if: matrix.os == 'windows-latest'
-
       - name: Install V
-        id: install-v
         uses: vlang/setup-v@v1.3
         with:
           check-latest: true
 
-      - name: Install via command (Linux and macOS)
-        if: matrix.os != 'windows-latest'
-        run: v -e "$(curl https://raw.githubusercontent.com/v-analyzer/v-analyzer/main/install.vsh -fsSL)"
+      - name: Install via webscript (Windows)
+        if: runner.os == 'Windows'
+        run: curl -o install.vsh https://raw.githubusercontent.com/v-analyzer/v-analyzer/main/install.vsh; v run install.vsh; del install.vsh
 
-      - name: Install via command (Windows)
-        if: matrix.os == 'windows-latest'
-        shell: msys2 {0}
-        run: v -e "$(curl https://raw.githubusercontent.com/v-analyzer/v-analyzer/main/install.vsh)"
+      - name: Install via webscript (Nix)
```

Comment:
Use `Unix`, or `nix` here. `Nix` unfortunately has a specific meaning these days (https://en.wikipedia.org/wiki/Nix_(package_manager)) , and can be confusing.

Reply:
Good point. Thanks!